tests/e2e/component.go

```diff
@@ -470,4 +472,228 @@ func componentTests(componentCmdPrefix string) {
 			waitForDeleteCmd("odo project list", projName)
 		})
 	})
+
+	var _ = Context("when component is in the current directory", func() {
```

Comment:
this should have some meaningful name, like `Testing odo push with --source and --config flags`